src/ens-avatar/README.md

```diff
@@ -0,0 +1,88 @@
+# ens-avatar
```

Comment:
+1

Reply:
Yeah, this implementation right now is a bit specific to our use-case (e.g. we can't perform arbitrary http requests since we have a confined xhr domain whitelist) which is why I didn't move it out into it's own package yet. Ideally, I'd like to contribute back to the `@ens-domains/ens-avatar` repo to resolve this, and also make the lib compatible with React Native, so maybe we can leave as-is for now until then?